src/mlpack/tests/split_data_test.cpp

```diff
@@ -108,6 +108,21 @@ BOOST_AUTO_TEST_CASE(SplitDataResultMat)
   CheckMatEqual(input, concat);
 }
 
+BOOST_AUTO_TEST_CASE(SplitDataResultMat)
+{
+  mat input(2, 10);
+  size_t count = 0; // count for putting unique sequential values
```

Comment:
Hm, should this be "counter for unique sequential values"?

Reply:
Done